java/impl/src/main/java/net/sf/mmm/code/impl/java/JavaRootContext.java

```diff
@@ -201,10 +201,4 @@ public static JavaRootContext get() {
     return instance;
   }
 
-  @Override
-  public ClassLoader getClassLoader() {
-
-    return this.loader.getClassLoader();
```

Comment:
With the latest version of the API, can you explain me how can I get the class loader?